frigate/test/test_camera_pw.py

```diff
@@ -8,7 +8,7 @@
 class TestUserPassCleanup(unittest.TestCase):
     def setUp(self) -> None:
         self.rtsp_with_pass = "rtsp://user:password@192.168.0.2:554/live"
-        self.rtsp_with_special_pass = "rtsp://user:password#$@@192.168.0.2:554/live"
+        self.rtsp_with_special_pass = "rtsp://user:password`~!@#$%^&*()-_;',.<>:\"\{\}\[\]@192.168.0.2:554/live"
```

Comment:
I'd like to keep the last character of the password as `@` in the test so we can be sure it won't interfere with the required @ in the rtsp url

Reply:
I updated the test to have another `@` at the end of the password.